Source/Manager/IdentityManager.swift

```diff
@@ -381,11 +381,21 @@ public class IdentityManager: IdentityManagerProtocol {
             completion(result)
         }
 
-        guard case .email = username else {
+        guard case .email(let email) = username else {
             wrappedCompletion(.failure(ClientError.unexpectedIdentifier(actual: username, expected: "only EmailAddress supported")))
             return
         }
 
+        SecAddSharedWebCredential(clientConfiguration.serverURL.host! as CFString,
```

Comment:
* This is triggered even when the user enters the wrong password.
  The credentials (i.e. username+password) are verified in the request just below this (`api.requestAccessToken`), so I think this should be placed in the async callback of that. 
* For initial release I'd like to see this feature wrapped behind a feature flag added in `IdentityUIConfiguration` (similar to what we have for use of touch/face id). It would need to be propagated from `PasswordCoordinator.submit` -> `SigninInteractor.login` -> `IdentityManager.login`

So something like:
  ```swift
  api.requestAccessToken(clientID: clientConfiguration.clientID,
                         ...
        ) { [weak self] result in
            switch result {
            case .success(_): // here we know the user credentials were correct
                if (storeInSharedWebCredentials) {
                    SecAddSharedWebCredential(...) 
                }
            default:
                break // do nothing
            }
            self?.finishLogin(result: result, persistUser: persistUser, completion: wrappedCompletion)
        }
```

Reply:
@zamzterz done. And I agree, having a feature flag like `useBiometrics` is a good call, so it's up to the consumers of the SDK to decide whether they want to enable it or not.